### PR TITLE
MTE-2178: New Homepage

### PR DESCRIPTION
**Pull request by brantdarilek**

### CODE REVIEW

**Comment by brantdarilek** [comment]
[run all]

File: styleguide/_themes/derek/scss/snowflakes/homepage.scss [hunk 1]
```diff
@@ -95,6 +95,42 @@
...
+
+.rsHomepage-subHeader-062018 {
+  color: $white;
+  font-size: 3.25rem;
+  line-height: 3.25rem;
+}
+
+.rsHomepage-button-062018 {
+  &.button {
+    &.ghost {
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/snowflakes/homepage.scss]: So this gets output as `.rsHomepage-button-062018.button.ghost`, which I think is one selector more than necessary here.  I think `.rsHomepage-button-062018.ghost` is sufficient as it occurs higher in the document than `.button.ghost` which we're trying to override.

Comment (WidgetsBurritos) [reply styleguide/_themes/derek/scss/snowflakes/homepage.scss]: Sorry, I meant lower, now higher.

**Comment by runrog** [comment]
👍 LGTM

**Comment by brantdarilek** [comment]
[run all]

**Pull request merged**